mods/cnc/chrome/lobby.yaml

```diff
@@ -6,13 +6,14 @@ Container@SERVER_LOBBY:
 	Height: 575
 	Children:
 		ColorPreviewManager@COLOR_MANAGER:
-		Label@SERVER_NAME:
+		LabelWithSelection@SERVER_NAME:
 			Width: PARENT_RIGHT
-			Y: 0 - 22
+			Y: 0 - 44
+			Height: 44
 			Font: BigBold
-			Contrast: true
 			Align: Center
 		Background@bg:
+			ClickThrough: true
```

Comment:
I'd prefer it if we didn't do this. I think #17684 should fix the underlying issue properly?